pom.xml

```diff
@@ -94,12 +94,6 @@
             <version>1.6.6</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.powermock</groupId>
-            <artifactId>powermock-module-junit4</artifactId>
-            <version>1.6.6</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
```

Comment:
I believe we don't use powermock-api-mockito either. If to clean things up, please remove that one either.

Reply:
done. thx for pointing out.